.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,10 @@
+| Q             | A
+| ------------- | ---
+| Bug fix?      | yes/no
+| Tests pass?   | yes                             <!-- please add some, will be required by reviewers -->
```

Comment:
This is rather redundant (red builds won't be merged).

Reply:
> This is rather redundant (red builds won't be merged).

I believe this is just a reminder to add some tests to cover the new feature or to avoid regression for a bug fix.

Reply:
Before dropping it completely what do you think about:

| Q | A |
| Test need? | yes/no